__tests__/Result_test.re

```diff
@@ -1,13 +1,13 @@
 open Jest;
 open Expect;
-open Rebase;
+open !Rebase;
```

Comment:
I suspect this should be `open! Rebase`. I wasn't aware that this was even valid syntax! `open!` shouldn't be used here though, it should either be opened normally or not at all.

Reply:
Reverted.